dlrover/python/master/elastic_training/rdzv_manager.py

```diff
@@ -198,7 +211,15 @@ def join_rendezvous(
                 self._start_rdzv_ts = time.time()
             if node_rank in self._waiting_nodes:
                 return self._rdzv_round
-            self._waiting_nodes[node_rank] = local_world_size
+            asw, psw = self._topology_querier.query(node_ip)
+            meta = NodeTopologyMeta(
```

Comment:
Better to distinguish the naming of local variables 'meta'. Consider there is so many concepts of metadata.